src/pages/userProfile.js

```diff
@@ -40,25 +41,65 @@ class Profile extends React.Component {
             console.log("From userProfile.js - error in getting the user's information", err);
         }
 
-        // Attempt to add one applied job for testing
+        // What to pass in when creating applied job objects. The id parameter must be different each time
+        const appliedJobInput = {
+            id: "54160v2d-7469-411e-9b53-012ed15c97d0",
+            dateApplied: "right this MOMENT",
+            status: "got it YET AGAIN",
+            appliedJobEmployeeId: "59560b0d-6069-431e-9b33-079ed12c08f0",   // my user id
+            appliedJobJobId: "54563b0d-7069-411e-9n33-049ed15c02f0"
+        }
+
+        const postedJobInput = {
+            id: "54563b0d-7069-411e-9n33-049ed15c02f0",
+            jobTitle: "Front End Developer",
+            jobType: "Management"
+        }
```

Comment:
There is no need for this, if you need to add data to the dynamodb, you should go to the aws console and do it by writing graphql.